python/cuml/test/test_mbsgd_classifier.py

```diff
@@ -24,18 +24,23 @@
 from sklearn.model_selection import train_test_split
 
 
-@pytest.mark.parametrize('lrate', ['constant', 'invscaling', 'adaptive'])
+@pytest.mark.parametrize(
```

Comment:
Please can you add `hinge-l2-invscaling`  to the list of inputs as this is the combination for which the stress test crashes without a stack trace. Issue #1942 mentions this. I think it is a good idea to have this combination in till we solve the issue.

Reply:
Good point! Added it.